db/migrate/20190730162819_create_uphold_connection_for_channel.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUpholdConnectionForChannel < ActiveRecord::Migration[5.2]
+  def change
+    create_table :uphold_connection_for_channels, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.belongs_to :uphold_connection, type: :uuid, index: true, null: false
+      t.belongs_to :channel, type: :uuid, index: true, null: false
+
+      t.string :currency, index: true
+      t.string :channel_identifier, index: true
```

Comment:
It doesn't seem like we really use this except for lookups in the future, right? Might be worth removing or mentioning that it's de-normalized. Might be worth having a conversation related https://stackoverflow.com/questions/5437744/naming-conventions-for-non-normalized-fields

Reply:
I've not really given much thought to the naming that much for these denormalized fields. 

Would you like to rename this to `derived_channel_identifier`?